src/CardComponent.js

```diff
@@ -0,0 +1,14 @@
+const CardComponent=(props)=>{
+    debugger;
+    return (
+        <div id="card" className="cardClass">
+            <img src={props.teamMembers.img}/>
```

Comment:
this is also good, but you can destruct them at the top,It will make your code ease.

Reply:
Implemented as per your comments